documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,465 @@
+// Do we have public links to these documents?
+// Additional RH-SDL resources:
+// * link:https://docs.google.com/document/d/1QMrM5ac2sbecmy7lYHA8S6p8L8ivVwHlgdcspy-Z4VE/edit#heading=h.66y4kqbj468a[Red Hat Secure Development Lifecycle Implementation Plan]
+// * link:https://docs.google.com/presentation/d/1CnIq-MHgEoq_1QgaFU5uoOfZ7ZOnNzxPk9OdDUe4Me8/edit#slide=id.g1a5a54f838a_0_1509[Red Hat Secure Development Lifecycle Introduction]
+// * Link:https://docs.google.com/presentation/d/19H3tSzZ1pSGGwhSoZn3CFgyLQcBWbePAK0_5J4NHUGw/edit#slide=id.g22dc74ad918_0_740[Red Hat Secure Development Lifecycle Planning and Schedule]
+// * link:https://docs.google.com/presentation/d/1DOxSd5hpwNntypX5DUd3JRmP8wIJL_RVxOJfDo0Nxck/edit#slide=id.g13028f60288_0_0[Secure Development - Introduction to SSML]
+// * link:https://gitlab.cee.redhat.com/users/auth/geo/sign_in[Closed link]
+// * link:https://docs.engineering.redhat.com/display/PRODSEC/Secure+Development+training[Secure Development Training]
+
+.{ocp} Provider
+
+This is a collection of new features for {ocp-name} (OCP) provider. link:https://issues.redhat.com/browse/MTV-739[(MTV-739)]
+
+.Block migrations from {ocp-name} versions before 4.13
+
+VM manifestations were introduced in {ocp-name} 4.13, so source provider clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+.Secure connections to remote {ocp-name} clusters
+
+Connections to remote {ocp-name} clusters can be secured with CA certificates. link:https://issues.redhat.com/browse/MTV-728[MTV-728]
+
+.Source and storage endpoints are not selected automatically
+
+Source network and source storage are not seleted automatically when creating an {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
+
+.Console Plugin
+
+This is a collection of new features for the {project-short} Console plugin. link:https://issues.redhat.com/browse/MTV-736[(MTV-736)]
+
+See descriptions for the following features that are included under Cross-Provider Features:
+
+* Replaced Forms and Wizards with {ocp-name} Console Create and Edit pages link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+* Restored vSphere Verify certificate functionality link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+* Tooltip explains the select migration network button link:https://issues.redhat.com/browse/MTV-245[(MTV-245)]
+
+* {ocp-name} Console native SDK fetch, post, and delete calls link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
+
+* Removed the Cancel icon during the Cutover stage link:https://issues.redhat.com/browse/MTV-449[(MTV-449)]
+
+* {project-short} UI plugin uses only a dynamic plugin for SDK REST API fetch methods link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
```

Comment:
```suggestion
* {project-short} UI plugin uses only a dynamic plugin for the SDK REST API fetch methods link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
```
* `fetch` methods
* Check whether method is plural or singular

Reply:
corrected
Checking with Arik